de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/configuration/RepositoryConfiguration.java

```diff
@@ -139,4 +140,34 @@ public String getLocalPath() {
 	public void setLocalPath(String localPath) {
 		properties.setProperty(LOCAL_PATH_KEY, localPath);
 	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;

```

Comment:
Codecoverage seems to be complaining about complete lack of coverage of hashCode() and also partially of equals()

Reply:
I would rather ignore it than put unnecessary code in tests to cover this auto-generated `hashCode()`

Reply:
Well if hashcode and equals is manually implemented, I think it should be tested.